snippets/mapped-table.md

```diff
@@ -0,0 +1,35 @@
+### MappedTable
+
+Generate a table with dynamic rows based on passed array of data. Optionally include headers.
```

Comment:
How about:

*Renders a table with rows dynamically created from an array of data.*

And then in the below explanation, add a line that states *Omit `headers`... blah blah*, like we usually do in 30code?